May to shall be able to (#238)
rocket-pool/RPIPs: RPIPs/RPIP-44.md

  - This function SHALL either remove the specified validators from the megapool or revert
      - If any of the validators are still active on the beacon chain a successful call of this
        function MUST also exit those validators from the beacon chain

>   - The Node Operator SHALL be able  to call this function (as implied by [RPIP-43](RPIP-43.md))

Reviewer: ```suggestion
  - The Node Operator SHALL be able to call this function (as implied by [RPIP-43](RPIP-43.md))
```

Reviewer: Remove double-space

>   - Anyone SHALL be able to call this function under the condition: `deficit >= exit_deficit`

Reviewer: ```suggestion
  - This function MUST be permissionlessly callable under the condition: `deficit >= exit_deficit`
```

Reviewer: Reword. Not sure if better. Is permissionlessly a word?
